### ISSUE TITLE
[high] JAWS+Chrome: Preferences: The toggle button cannot be activated with enabled screen reader

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5530]
## Action Performed:
Prerequisite: the user is logged in.
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Navigate to Settings > Preferences.
3. Once the Preferences overlay opens, press Tab key to focus the switch button 'Receive relevant feature updates and Expensify news'.
4. Try to activate the element by pressing Space or Enter keys.

## Expected Result:
The toggle button should be accessible when pressing Space or Enter keys with enabled JAWS.

## Actual Result:
The toggle button cannot be activated with Space or Enter with enabled JAWS.
Note: it works fine when the screen reader is disabled.

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1777fci r-1otgn73 r-1i6wzkk r-lrvibr" style="background-color: rgb(3, 212, 124); border-radius: 20px; height: 28px; padding: 15px; transition-duration: 0s; width: 50px;"><div class="css-1dbjc4n r-u8s1d" style="background-color: rgb(255, 255, 255); border-radius: 11px; height: 22px; left: 4px; transform: translateX(20px); width: 22px;"></div></div>`

## Workaround:
No workaround

## Area issue was found in:
Preferences overlay

## Failed WCAG checkpoints
2.1.1

## User impact:
The screen reader users are not able to interact with the element.

## Suggested resolution:
Implement the element using native `<button>` as suggested in issue 5183741 to make it accessible by default. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes

https://user-images.githubusercontent.com/88733897/134781252-52d5f42e-383b-412a-9f6b-242a6bff5746.mp4

![Bug5183750_keyboard_-_toggle_button_-_preferences](https://user-images.githubusercontent.com/88733897/134781254-4c2f96d2-3c0f-4775-aae5-c61f78df7608.png)

**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
